### DISCUSSION TITLE
server recommendation for Frigate

### DISCUSSION THREAD

**Discussion by EduardoHerreraCarrillo** [blakeblackshear/frigate#8575] [General]
Dear All:
Along with greeting you, I write to ask a question.
I need to connect 48 cameras and I would like to know if I can manage them with FRIGATE.
If possible, what are the hardware requirements that a server must have to set up FRIGATE?

**Comment by NickM-27** [comment]
there was a similar question at https://github.com/blakeblackshear/frigate/discussions/7491

**Reply by EduardoHerreraCarrillo** [reply]
thank you very much, I will check
